.gitignore

```diff
@@ -0,0 +1 @@
+aritGeo/node_modules/*
```

Comment:
@Billmike Always add a new line after the end of a file.

Reply:
@koyexes Alright. Noted.